### PR TITLE
Extract Climate Information from the EIFFEL Ontology to be added to the DKG

### PR DESCRIPTION
**Pull request by nanglo123**
- [x] Extract information and relationships from four of the nine turtle files listed in the EIFFEL Ontology
- [x] Create a list of terms from each result in a query
- [x] Preprocess queried label and description strings  
- [x] Add the information to the DKG

Demo after running:

http://0.0.0.0:8771/api/entity/eotaxonomy:BioDiversityAndLandEcosystemArea gives the following:

```json
{
  "id": "eotaxonomy:BioDiversityAndLandEcosystemArea",
  "name": "BioDiversity and Land Ecosystem",
  "type": "class",
  "obsolete": false,
  "description": "…data assimilation techniques to combine satellite observations of the land surface with radiative transfer and other physical models to provide best estimates of biophysical parameters, improving understanding of ecosystems and ecosystems and assessing vulnerability, etc.",
  "synonyms": [],
  "xrefs": [],
  "labels": [
    "eotaxonomy"
  ],
  "properties": {}
}
```

### CODE REVIEW

File: mira/dkg/resources/eiffelnames.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
+"""Get terms from eiffel names."""
```
Comment (cthoyt) [inline mira/dkg/resources/eiffelnames.py:1]: is this file needed?

File: mira/dkg/resources/extract_eiffel_ontology.py [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+        file.write(
+            f'Number of unique predicates: {len(unique_predicates)}\n\n')
+
+        # Write each unique predicate to the file
+        for predicate in unique_predicates:
+            file.write(f'{predicate}\n')
+
+
+def process_eo(graph_obj, converter):
+    info_query = """
```
Comment (cthoyt) [inline mira/dkg/resources/extract_eiffel_ontology.py:154]: save all of these queries as top-level constants so as to declutter the functions

File: mira/dkg/resources/extract_eiffel_ontology.py [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+              '/sdg-kos-goals-targets-indicators.ttl',
+              'w') as file:
+        file.write(sdg_goal_request.text)
+
+    graph_obj_map = {}
+    for file_name in os.listdir(ONTOLOGY_FILES_DIR):
+        content_path = 'eiffel_ttl_files/' + file_name
+        with open(content_path) as file:
+            graph_obj_ = GraphObj(graph=Graph(),
+                                  name=file_name[:len(file_name) - 4])
```
Comment (cthoyt) [inline mira/dkg/resources/extract_eiffel_ontology.py:61]: when you use the `pathlib` interface, you get lots of nice high-level operations so you don't have to re-implement filename parsing yourself

File: mira/dkg/resources/extract_eiffel_ontology.py [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+            graph_obj_.graph.parse(file, format='turtle')
+            graph_obj_map[file_name] = graph_obj_
+    return graph_obj_map
+
+
+def process_ecv(graph_obj, converter):
```
[the remark this answers is not in the document]
Comment (nanglo123) [reply mira/dkg/resources/extract_eiffel_ontology.py]: graph_obj isn't actually an rdflib.Graph object but an object built from a custom class I defined to store information from queries to write to csv files

File: mira/dkg/resources/eiffelnames.py (hunk 1 above)
Comment (nanglo123) [reply mira/dkg/resources/eiffelnames.py:1]: This file I created to create terms for eiffel

**Review by cthoyt — APPROVED** [review]
Looking good, great work @nanglo123
